modules/gis/test/reproject.spec.js

```diff
@@ -0,0 +1,62 @@
+import test from 'tape-promise/tape';
+import {mapBinaryCoords, mapGeoJsonCoords} from '@loaders.gl/gis';
+import {Proj4Projection} from '@math.gl/proj4';
+
+test('gis#reproject GeoJSON', t => {
+  const projection = new Proj4Projection({from: 'WGS84', to: 'EPSG:3857'});
+  const inputGeoJson = [
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [-74, 41]
+      },
+      properties: {}
+    }
+  ];
+  const expectedGeoJson = [
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [-8237642.318702244, 5012341.663847514]
+      },
+      properties: {}
+    }
+  ];
+
+  const out = mapGeoJsonCoords(inputGeoJson, coord => projection.project(coord));
+  t.deepEqual(out, expectedGeoJson);
+  t.end();
+});
+
+test('gis#reproject binary', t => {
+  const projection = new Proj4Projection({from: 'WGS84', to: 'EPSG:3857'});
+  const binaryData = {
+    points: {
+      positions: {value: new Float32Array([-74, 41]), size: 2},
+      globalFeatureIds: {value: [0, 1, 1], size: 1},
+      featureIds: {value: [0, 1, 1], size: 1},
+      numericProps: {
+        numeric1: {value: [1, 2, 2], size: 1}
+      },
+      properties: [{string1: 'a'}, {string1: 'b'}]
+    }
+  };
+  const expectedBinaryData = {
+    points: {
+      positions: {value: new Float32Array([-8237642.318702244, 5012341.663847514]), size: 2},
+      globalFeatureIds: {value: [0, 1, 1], size: 1},
+      featureIds: {value: [0, 1, 1], size: 1},
+      numericProps: {
+        numeric1: {value: [1, 2, 2], size: 1}
+      },
+      properties: [{string1: 'a'}, {string1: 'b'}]
+    }
+  };
+
+  const out = mapBinaryCoords(binaryData, coord => projection.project(coord));
```

Comment:
We should bind the `project` and `unproject` functions in math.gl so we don't need helper functions here.

```js
const out = mapBinaryCoords(binaryData, projection.project);
```

Reply:
Ref https://github.com/uber-web/math.gl/pull/163